_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,300 @@
+
+> I love how you’ve contributed back to the open source community. I am a strong believer in the power of open source, especially in the iOS community, where there has historically been a resistance to opening software. Sponsoring CocoaPods is a fantastic contribution toward the iOS community and I thank you for it. I’ve contributed back to several projects on GitHub, have written for the Teehan+Lax blog professionally, and have a selection of the apps I’ve written on my portfolio.
+
+I know what you're thinking "Ash wasn't a junior when he applied", well, chances are you've still got a history of things that can be applied towards an email like this. Also, flattery can get you everywhere. "I've been using your app for years, I love how it does 'x' - have you thought of doing 'y'?"
+
+Sarah's was similar:
+
+> First, I'd just like to say I love Artsy and have been following your company since I learned of its existence two years ago. Making art accessible to the world is a problem the architects of the Internet should undoubtedly be solving and Artsy is doing so with elegance.
+
+_10/10_ ✔ 
+
+> I'm currently transitioning from embedded systems to iOS development and looking for a place where I can learn a lot while making significant contributions to a product. As for my background, I just graduated from NYU with a B.S. in Electrical Engineering and completed a seven-month-long research and development internship at Canary (a home security hardware startup). While I do enjoy building hardware, I find software engineering generally more fun and in tune with my intellectual and creative preferences.
+
+Dead on. Later on in the email, she references [art critics](https://en.wikipedia.org/wiki/Rosalind_E._Krauss) while discussing [programming concepts](https://en.wikipedia.org/wiki/Enumeration). A great introduction email.
+
+### On a Portfolio
+
+_on wording_: I use the word portfolio, to encapsulate a CV/Resume/Design Portfolio. They all have [specific meanings](http://www.diffen.com/difference/Curriculum_Vitae_vs_Résumé), but in this case, I mean something you attach to the email to provide the full context of your history/experiences.
+
+There are a lot of places for good, solid advice on the document you are using to persuade someone to interview you. So I'll tell you what has worked for me. I think it should be a one page document, that captures a snapshot of you. Things they must have: your name, a way to contact you and a list of things you think is relevant.
+
+You should consider what you think are qualities that you bring to the table, for some job applications I have submitted both a resume and a design portfolio. If you are particularly proud of your design work, perhaps find a way to include your app store screenshots and branding, or make your portfolio distinctive via design.
+
```

Comment:
Not sure if it's worth adding here somewhere, or if you'd like to leave it to the advice in the links, but the advice I've always given people is that your cv/portfoilo/resume/thing needs to be incredibly readable for others. Too often have I read and seen CVs that were just so full of text I could barely make it through, even though I actually _wanted_ to, let alone if I was just scanning it. 

Maybe something like "Someone else is going to have to read and understand your relevant history, so spend some time on making it as readable and easily digestible as possible." ?